test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -51,12 +51,20 @@ FAILED_TESTS=""
 export OVERRIDE_BAZEL_VERSION="$VERSION"
 # when running under bazel docker image, the workspace is read only.
 export OVERRIDE_BAZEL_WRAPPER_DOWNLOAD_DIR=/tmp
-bazel build -- //... "${EXCLUDED_TARGETS[@]}" || FAILED_TESTS="${FAILED_TESTS}Build "
+
+ACTION_ENV_FLAG=""
+if [ ! -z "$KOKORO_ARTIFACTS_DIR" ]; then
```

Comment:
seems like you don't need the check that you're on kokoro anymore? (I assume that's what `! -z "$KOKORO_ARTIFACTS_DIR"` is supposed to mean.

Reply:
always setting ACTION_ENV_FLAG will also help you avoid the shellcheck error (see sanity test failure).